nconnect.go

```diff
@@ -310,9 +328,11 @@ func (nc *nconnect) getRemoteInfo(remoteAdminAddr string) (*admin.GetInfoJSON, e
 }
 
 func (nc *nconnect) StartClient() error {
-	err := nc.opts.VerifyClient()
-	if err != nil {
-		return err
+	if !nc.opts.NetworkMember {
```

Comment:
I would say we put this if inside VerifyClient()

Reply:
VerifyClient is a method of Config, and it can't get the mode(client, server, or network) of nConnect. An alter way is to put VerifyClient into Opts.